pkg/tsdbctl/add.go

```diff
@@ -248,6 +252,10 @@ func strToLabels(name, lbls string) (utils.Labels, error) {
 			if len(splitLbl) != 2 {
 				return nil, errors.New("labels must be in the form: key1=label1,key2=label2,...")
 			}
+
+			if err := utils.IsValidLabelName(splitLbl[0]); err != nil {
+				return nil, errors.Wrap(err, fmt.Sprintf("illegal lable name: '%s'", splitLbl[0]))
```

Comment:
Typo. Do we really need to wrap it anyway? There's no information here that's not already in `err`.

Reply:
Wrapping it to create better stack trace.